playground/app.vue

```diff
@@ -8,12 +8,16 @@
       <p class="text-gray-500 dark:text-gray-400">
         Try your components here!
       </p>
+      <p>Count: {{ count }}</p>
```

Comment:
Can you remove the changes in the playground?

Reply:
Forgot those, done in 8d5721f